src/elm/Routes/DetailedQuotePage.elm

```diff
@@ -9,12 +9,12 @@ import Types exposing (..)
 
 detailedQuotePage : Model -> Html Msg
 detailedQuotePage model =
-    div [ class "mb4" ]
-        [ header [ class "header pl3" ]
-            [ h1 [ class "tl f3 fw7 gray mb1" ] [ text "Women like you" ]
-            ]
+    div [ class "ma4 ml5-ns mr5-ns" ]
+        [ div [ class "gray f3 b" ] [ text "Women like you" ]
+        , div [ class "gray fw1 mt3 mb3 mb4-ns w-50-l" ] [ text "Below is a real story from someone we've helped. We provide all the services that they used. If you think something they've used would help you as well or if you're still not sure please get in touch with us." ]
+        , div [ class "purple-line mb4" ] []
         , retrieveTestimonial model
-        , purpleButton "See full list of services" "services"
-        , purpleButton "Contact us" "talk-to-us"
+        , div [ class "dn-ns" ] [ purpleButton "See our services" "services" ]
+        , div [ class "dn-ns" ] [ purpleButton "Contact us" "talk-to-us" ]
```

Comment:
Do we want these `ns` or `l` - at the moment everything else (like the navbar) changes using the `l`/large flag

Reply:
realised you might have done this on purpose, but will wait till tomorrow to merge

Reply:
Now complete